### PR TITLE
Update MetricsMonitorTest.java

### PR DESCRIPTION
**Pull request by Erikkre**
#7274 
#build

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/MetricsMonitorTest.java [hunk 1]
```diff
@@ -117,7 +117,7 @@ public void testMpMetrics10Monitor10Feature() throws Exception {
     	Log.info(c, testName, "------- Enable mpMetrics-1.0 and monitor-1.0: vendor metrics should not be available ------");
     	server.setServerConfigurationFile("server_mpMetric10Monitor10.xml");
     	server.startServer();
-    	Log.info(c, testName, server.waitForStringInLog("defaultHttpEndpoint-ssl",60000));
+    	Log.info(c, testName, server.waitForStringInLog("SRVE9103I",60000));
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/MetricsMonitorTest.java]: If SRVE9103I is not found, the test will continue.  Should all of these `server.waitForStringInLog()` be wrapped with `Assert.assertNotNull()`?

Comment (Erikkre) [reply dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/MetricsMonitorTest.java]: I've put another commit on the branch with the assert.

**Comment by Erikkre** [comment]
#build

**Comment by Erikkre** [comment]
Fixes #7274

**Pull request merged**